gramps/plugins/tool/reorderids.py

```diff
@@ -34,207 +35,516 @@
 from gramps.gen.const import GRAMPS_LOCALE as glocale
 _ = glocale.translation.gettext
 
+from gi.repository import Gtk, Gdk
+
 #------------------------------------------------------------------------
 #
-# Gramps modules
+# GRAMPS modules
 #
 #------------------------------------------------------------------------
-from gramps.gui.utils import ProgressMeter
-from gramps.gen.lib import (Event, Family, Media, Note,
-        Person, Place, Repository, Source, Citation)
+from gramps.gen.const import URL_MANUAL_PAGE
+
 from gramps.gen.db import DbTxn
-from gramps.gui.plug import tool
+from gramps.gen.lib import (Person, Family, Event, Place,
+                            Source, Citation, Repository, Media, Note)
 
-_findint = re.compile('^[^\d]*(\d+)[^\d]*')
+from gramps.gui.display import display_help
+from gramps.gui.glade import Glade
+from gramps.gui.managedwindow import ManagedWindow
+from gramps.gui.plug import tool
+from gramps.gen.updatecallback import UpdateCallback
+from gramps.gui.widgets import MonitoredCheckbox, MonitoredEntry
 
-# gets the number specified in a format string, example: %04d returns '04'
-_parseformat = re.compile('.*%(\d+)[^\d]+')
+#-------------------------------------------------------------------------
+#
+# Constants
+#
+#-------------------------------------------------------------------------
+WIKI_HELP_PAGE = '%s_-_Tools' % URL_MANUAL_PAGE
+WIKI_HELP_SEC = _('manual|Reorder_GRAMPS_ID')
```

Comment:
This should be **Reorder_Gramps_ID** as it will go to the Gramps 5.0 version of this url: https://gramps-project.org/wiki/index.php?title=Gramps_4.2_Wiki_Manual_-_Tools#Reorder_Gramps_ID